### DISCUSSION TITLE
Export/Import ui settings Vue 3

### DISCUSSION THREAD

**Discussion by Beet4** [VueTorrent/VueTorrent#1318] [General]
Hi! Just wanted to ask if you have plans to bring back the future to export/import the UI settings that we had in the Vue 2 version? It was nice not having to redo my torrent card settings on all devices. :)

I know you have lots to do in Vue 3 version so not trying to push or anything, just wanted to ask. :)

**Comment by Larsluph** [comment]
It's not planned for now.
We removed it because with the Vue3 upgrade we changed the way we handle localStorage. No ETA on when it will be back.

See #1322 to track when it will be released.
